substratevm/src/com.oracle.svm.hosted/src/com/oracle/svm/hosted/SecurityServicesFeature.java

```diff
@@ -278,11 +278,18 @@ private static void linkJaas(DuringAnalysisAccess duringAnalysisAccess) {
             try {
                 /*
                  * Access the Provider.knownEngines map and extract the EngineDescription
-                 * corresponding to the serviceType. From the EngineDescription object extract the
-                 * value of the constructorParameterClassName field then, if the class name is not
-                 * null, get the corresponding Class<?> object and return it.
+                 * corresponding to the serviceType. Note that the map holds EngineDescription(s) of
+                 * only those service types that are shipped in the JDK. From the EngineDescription
+                 * object extract the value of the constructorParameterClassName field then, if the
+                 * class name is not null, get the corresponding Class<?> object and return it.
                  */
                 /* EngineDescription */Object engineDescription = knownEngines.get(serviceType);
+                // this isn't a engine known to the Provider (which actually means that it isn't one
+                // that's shipped in the JDK), so we don't have the predetermined knowledge of the
+                // constructor param class
```

Comment:
Nitpicking, but can you please use the `/**/` style for the above comment. Also, use capitalization and end it with a `.`. We like to keep our comment style consistent as much as possible. :)

Reply:
Done :)